ebu_tt_live/node/delay.py

```diff
@@ -42,7 +43,20 @@ def process_document(self, document, **kwargs):
                 # change the sequence identifier
                 document.sequence_identifier = self._document_sequence
 
-                # TODO: add an ebuttm:appliedProcessing element to the document metadata
+                if document.binding.head.metadata is None:
+                    document.binding.head.metadata = metadata.headMetadata_type(
+                        metadata.documentMetadata()
+                    )
+
+                if document.binding.head.metadata.documentMetadata is None:
+                    document.binding.head.metadata.documentMetadata = metadata.documentMetadata()
+
+                document.binding.head.metadata.documentMetadata.appliedProcessing = metadata.appliedProcessing_type(
```

Comment:
What happens if there is already an appliedProcessing element? I did a test and found that it gets overwritten. In fact when the XSD is updated to allow any number of appliedProcessingElements this line fails - some other way to append the element is needed, which I have not yet worked out how to do...